Actions/AL-Go-Helper.ps1

```diff
@@ -666,6 +666,8 @@ function ReadSettings {
             "Account"                                   = ""
             "CertificateProfile"                        = ""
         }
+        "useSubModules"                                 = ""
+        "subModulesTokenSecretName"                     = "subModulesToken"
```

Comment:
Can't we just hardcode the secret name instead of it being a setting? Like the Azure_Credentials secret for example. That would save us from calling ReadSecrets before actions/checkout since we know what the secret name is

Reply:
All secrets (incl. azure_credentials) can be mapped to other secrets using the SecretName setting construct.
The default is subModulesToken, so people don't have to do anything and we could also remove this setting from AL-Go-Helper and it would still work. This setting is just to symbolize that it works like this

Reply:
and we also need to support getting this from an Azure KeyVault etc etc.